src/components/Navbar/Navbar.js

```diff
@@ -137,6 +198,55 @@ function Navbar() {
                   Sign-up
                 </button>
               </div>
+              )}
+              
+
+            
+            {login  && (
+
+              <div className="d-grid gap-2 d-md-flex justify-content-md-end">
+             
+            <img id="img-user"
+              src={require("../../assets/user-logo.png")}
+              alt="Logotipo Empresa"
+            />
+         
+              <li className="nav-item dropdown">
+                <a
+                  className="nav-link dropdown-toggle"
+                  href="/"
+                  role="button"
+                  data-bs-toggle="dropdown"
+                  id="text"
+                >
+                  {name}
+                </a>
+                <ul className="dropdown-menu">
+                  <li>
+                    <button
+                      onClick={UpdateUser}
+                      
+                      className="dropdown-item"
+                      id="text-dropdown"
+                    >
+                      Tu perfil
+                    </button>
+                  </li>
+                  <li>
+                    <button
+                      className="dropdown-item"
+                      id="text-dropdown"
+                      onClick={SignOut}
+                    >
+                      Cerrar sesión
+                    </button>
+                  </li>
+                
+                </ul>
```

Comment:
![image](https://user-images.githubusercontent.com/118463732/218143646-a74f6c7f-4edc-46d6-b170-7bc0a640bb8d.png)
Find a way to make the dropdown list open to the left